src/main/java/sg/dex/crypto/Hash.java

```diff
@@ -123,4 +127,34 @@ public static String sha3_256String(String string) {
         return Hex.toString(sha3_256(string));
     }
 
+    /**
+     * This method is used to compute the complete hash computation based on input stream passed.
+     * A MessageDigest object starts out initialized.
+     * The data is processed through it using the update methods.
+     * At any point reset can be called to reset the digest.
+     * Once all the data to be updated has been updated, one of the digest
+     * methods should be called to complete the hash computation.
+     *
+     * @param inputStream
+     * @return
+     */
+    public static String checkSum(InputStream inputStream) throws DigestException {
```

Comment:
Can we use a better name? It isn't really a checksum.

Reply:
update name as computeHashWithSHA3